thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/detection/trigger/utils/DataAvailabilitySchedulingConfiguration.java

```diff
@@ -37,6 +36,8 @@
   private long consumerPollTimeInMilli = 5_000; // consumer wait 5 secs by default for the buffer to be filled
   private List<String> dataSourceWhitelist;
   private List<String> filterClassList;
+  private long schedulerDelayPerRunInSec = 300;
+  private int globalMaxNotRunThresholdInMin = 24 * 60;
```

Comment:
1. Add comment on what it is.
2. rename to "taskTriggerFallBackTime"
3. Let's align everything in second